clients/drcachesim/tools/opcode_mix.cpp

```diff
@@ -252,7 +257,35 @@ opcode_mix_t::process_memref(const memref_t &memref)
 static bool
 cmp_val(const std::pair<int, int64_t> &l, const std::pair<int, int64_t> &r)
 {
-    return (l.second > r.second);
+    return (l.second > r.second) || (l.second == r.second && l.first < r.first);
+}
+
+std::string
+get_category_names(uint category)
+{
+    std::string category_name;
+    if (category == DR_INSTR_CATEGORY_UNCATEGORIZED) {
+        category_name += instr_get_category_name(DR_INSTR_CATEGORY_UNCATEGORIZED);
+        return category_name;
+    }
+
+    uint max_mask = 0x80000000;
```

Comment:
This could be a constant (either `const` or `#define`, and name in all-caps).

Reply:
Done.